Cargo.toml

```diff
@@ -74,7 +74,7 @@ petgraph = "0.6"
 predicates = "3"
 proc-macro2 = "1"
 quote = "1"
-reqwest = { version = "0.11.22", features = ["gzip", "brotli", "deflate", "json", "stream"] }
+reqwest = { version = "0.11.22", features = ["gzip", "brotli", "deflate", "json", "stream", "native-tls-vendored"] }
```

Comment:
OpenSSL vendoring has some tricky consequences that I believe we cannot afford, given the importance of SSL in our security.

As per https://docs.rs/openssl/latest/openssl/#vendored:

> If the vendored Cargo feature is enabled, the openssl-src crate will be used to compile and statically link to a copy of OpenSSL. The build process requires a C compiler, perl (and perl-core), and make. The OpenSSL version will generally track the newest OpenSSL release, and changes to the version are not considered breaking changes.

This means that if OpenSSL gets a critical vulnerability fix, it will be our responsibility to release Scarb and relay it further to users, instead of the users' OS.

> The vendored copy will not be configured to automatically find the system’s root certificates, but the openssl-probe crate can be used to do that instead.

This sounds like a feature regression that would need further fixing.

---

I wonder, can't we just install OpenSSL in Cross containers, following this instruction: https://github.com/cross-rs/cross/wiki/Recipes#pre-build ?